package/MDAnalysis/coordinates/memory.py

```diff
@@ -559,6 +560,9 @@ def timeseries(self, asel=None, start=0, stop=-1, step=1, order='afc'):
         if (asel is None or asel is asel.universe.atoms):
             return array
         else:
+            if len(asel) == 0:
+                raise NoDataError("Timeseries requires at least one atom "
```

Comment:
Same discussion as above.

Reply:
Make it `ValueError`.

Reply:
You'd still want to document it with a versionchanged and list it under fixes in CHANGELOG (we're fixing the meaning) — or listing it under Changes and pointing out that NoDataError was derived from ValueError??

Reply:
Thanks all, I will change ASAP, travelling this weekend